pkg/healthcheck/healthcheck.go

```diff
@@ -404,6 +415,9 @@ func (hc *HealthChecker) allCategories() []category {
 					description: "control plane namespace does not already exist",
 					hintAnchor:  "pre-ns",
 					check: func(context.Context) error {
+						if hc.NoInitContainer {
```

Comment:
now that we're installing the CNI daemonset into a separate namespace, we can remove this check, right?